packages/plugin-odoo/src/Odoo.ts

```diff
@@ -51,15 +52,51 @@ export class Odoo<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<T>
                 },
             },
             {
-                id: 'OdooColorPickerButton',
-                async render(): Promise<ActionableNode[]> {
+                id: 'OdooTextColorButton',
+                async render(): Promise<DividerNode[]> {
                     const button = new ActionableNode({
-                        name: 'colorpicker',
-                        label: 'Color picker',
-                        commandId: 'openColorPicker',
-                        modifiers: [new Attributes({ class: 'fa fa-eyedropper fa-fw' })],
+                        name: 'textcolorpicker',
+                        label: 'Text Color picker',
+                        commandId: 'openTextColorPicker',
+                        modifiers: [
+                            new Attributes({ class: 'fa fa-font fa-fw dropdown-toggle' }),
+                            new Attributes({ 'data-toggle': 'dropdown' }),
+                        ],
                     });
-                    return [button];
+                    const dropdownContent = new DividerNode();
+                    dropdownContent.modifiers.append(new Attributes({ class: 'dropdown-menu' }));
+                    const dropdownContainer = new DividerNode();
+                    dropdownContainer.modifiers.append(
+                        new Attributes({ class: 'dropdown jw-dropdown jw-dropdown-textcolor' }),
+                    );
+                    dropdownContainer.append(button);
+                    dropdownContainer.append(dropdownContent);
+                    return [dropdownContainer];
```

Comment:
The VNodes are intented to be "sementic" node, the one you add here are more what should be rendered out of the sementic nodes. You should make a renderer with a predicat for `ActionableNode` that is 'textcolorpicker' or 'textcolorpicker'.